src/components/timeline/Ruler.css

```diff
@@ -50,3 +50,14 @@
   right: 0;
   padding-right: 5px;
 }
+
+@media (forced-colors: active) {
+  /* The background gradient we use to display the notch are not visible in HCM.
+     We need to use a plain color, and we can't set the background color otherwise
+     it also covers the text */
+  .timelineRulerNotch {
+    /* In regular mode, this is 1px, so bump it down to account for the added border */
+    width: 0;
+    border-inline-end: 1px solid CanvasText;
```

Comment:
Using border instead of width also makes it consistent. With my zoom level, `width: 1px` doesn't make it always look like the same width due to rounding differences... but `border` is always consistent in this case.
This isn't that visible with the grey gradient (although now I'll see it always I guess :D), but with `CanvasText` this is very visible. I think we might want to use border-inline-end for the normal case too... (but this is out of scope for this patch).

Reply:
can you please add a comment explaining that borders round to integer pixels, but widths don't?

Reply:
I switched approach to use the background color and removed backplating on the span, adding a comment to explain why

Reply:
so that if we switch to using borders, it will be the same for HCM and non HCM